src/tbp/monty/frameworks/experiments/monty_experiment.py

```diff
@@ -328,7 +332,7 @@ def init_loggers(self, logging_config: dict[str, Any]) -> None:
         self.python_log_level = logging_config["python_log_level"]
         self.log_to_file = logging_config["python_log_to_file"]
         self.log_to_stderr = logging_config["python_log_to_stderr"]
-        self.output_dir = logging_config["output_dir"]
+        self.output_dir = Path(logging_config["output_dir"])
         self.run_name = logging_config["run_name"]
 
         if not os.path.exists(self.output_dir):
```

Comment:
Isn't there a `pathlib` version of this method on Path objects? Could fix it while you're here.